executor/aggfuncs/aggfunc_test.go

```diff
@@ -234,6 +234,31 @@ func distinctUpdateMemDeltaGens(srcChk *chunk.Chunk, dataType *types.FieldType)
 	return memDeltas, nil
 }
 
+func noZeroUpdateMemDeltaGens(srcChk *chunk.Chunk, dataType *types.FieldType) (memDeltas []int64, err error) {
```

Comment:
I think this function has something wrong. eg. The rowNum of buildAggMemTester is 100, the rows are `0, 1, ..., 99`, And the memDelta of the `Max` should be `1, 0, 0, 0, ...., 1, 0, 0, ..., 0`. Maybe you should customize a new MemDeltaGens function for you unit test args.